src/expressions/operator-expr.md

```diff
@@ -79,20 +81,18 @@ let a = && && mut 10;
 let a = & & & & mut 10;
 ```
 
-### Raw address-of operators
+### Raw pointer operators
 
-Related to the borrow operators are the *raw address-of operators*, which do not have first-class syntax, but are exposed via the macros [`ptr::addr_of!(expr)`][addr_of] and [`ptr::addr_of_mut!(expr)`][addr_of_mut].
+Related to the borrow operators are the *raw pointer operators*, `&raw const` and `&raw mut`.
 The expression `expr` is evaluated in place expression context.
```

Comment:
What `expr` is hasn't been specified (as the old text it was referring to has been removed).

```suggestion
The operand expression of the raw pointer operator is evaluated in place expression context.
```